### PR TITLE
2016 06 cc ui bits

### PR DESCRIPTION
**Pull request by localgrr**
Some UI stuff.
Ideas list had some weird text in it on staging
added the "Optionen" button instead of dots
added phase color coding works on both topics and ideas detail now

### CODE REVIEW

File: static-src/scss/core/_hero-unit.scss [hunk 1]
```diff
@@ -49,6 +49,38 @@
     .button-group {
         @include rem(margin-top, 20px);
     }
+
+    &.phase-PhaseRefinement {
```
Comment (fisx) [inline static-src/scss/core/_hero-unit.scss]: why wasn't this necessary for the topic overview page?

Comment (fisx) [inline static-src/scss/core/_hero-unit.scss]: ah, got it.  sorry.

(it was, but it needs to be in a more general place now.)

**Comment by fisx** [comment]
LGTM.  can be merged when green.

**Comment by andorp** [comment]
LGTM

**Pull request merged**